src/components/KeyboardShortcuts/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { ConnectedReduxProps } from '../../configureStore';
+import {
+  FileTree,
+  RelativePathPosition,
+  goToRelativeFile,
+} from '../../reducers/fileTree';
+import { actions as versionsActions } from '../../reducers/versions';
+import styles from './styles.module.scss';
+import { gettext } from '../../utils';
+
+const keys = ['k', 'j', 'e', 'o', 'c'];
+
+export type PublicProps = {
+  currentPath: string;
+  pathList: FileTree['pathList'];
+  versionId: number;
+};
+
+export type DefaultProps = {
+  _document: typeof document;
+  _goToRelativeFile: typeof goToRelativeFile;
+};
+
+type Props = PublicProps & DefaultProps & ConnectedReduxProps;
+
+export class KeyboardShortcutsBase extends React.Component<Props> {
+  static defaultProps: DefaultProps = {
+    _document: document,
+    _goToRelativeFile: goToRelativeFile,
+  };
+
+  keydownListener = (event: KeyboardEvent) => {
+    const {
+      _goToRelativeFile,
+      currentPath,
+      dispatch,
+      pathList,
+      versionId,
+    } = this.props;
+
+    if (
+      !event.altKey &&
+      !event.ctrlKey &&
+      !event.metaKey &&
+      !event.shiftKey &&
+      keys.includes(event.key)
+    ) {
+      switch (event.key) {
+        case 'k':
+          dispatch(
+            _goToRelativeFile({
+              currentPath,
+              pathList,
+              position: RelativePathPosition.previous,
+              versionId,
+            }),
+          );
+          break;
+        case 'j':
+          dispatch(
+            _goToRelativeFile({
+              currentPath,
+              pathList,
+              position: RelativePathPosition.next,
+              versionId,
+            }),
+          );
+          break;
+        case 'e':
```

Comment:
Maybe add a comment here explaining that it's from the old site? Otherwise it's not obvious why we are supporting it.